inst/book/more-reddim.Rmd

```diff
@@ -479,8 +480,22 @@ dm <- densmap(reducedDim(sce.zeisel, "PCA"), dens_frac = 0.4, dens_lambda = 0.2)
 reducedDim(sce.zeisel, "densMAP") <- dm
 ```
 
+
+```{r, echo=FALSE}
+ds <- reducedDim(sce.zeisel, "dens-SNE")
+ts <- reducedDim(sce.zeisel, "TSNE")
+
+vars_d <- colVars(ds[sce.zeisel$level1class == "astrocytes_ependymal", ])
+vars_t <- colVars(ts[sce.zeisel$level1class == "astrocytes_ependymal", ])
+stopifnot(mean(vars_d) > mean(vars_t))
```

Comment:
Are the two sets of coordinates comparable on the same scale? Should we normalize the variances by, e.g., the range of the plot?